cli/flox/src/commands/services/mod.rs

```diff
@@ -200,6 +183,12 @@ pub async fn start_with_new_process_compose(
     Ok(names)
 }
 
+/// Warning to print when there are changes to the manifest that might require
+/// restarting services.
+pub fn manifest_has_changes_for_services_warning() -> String {
+    "Your manifest has changes that may require running 'flox services restart'.".to_string()
+}
+
```

Comment:
suggestion - nb: make `const`
since this is not doing any interpolation anymore, i'd say its now effectively a constant.